File: project-Euler/python/2.py
def fiboEvenSum(num):
    a = 1
    b = 2
    sum = b
    next = a+b
    while next<= num:
        if next % 2 == 0:
            sum+=next
            temp = next
            next+=b
            b = temp
        else:
            temp = next
            next+=b
            b = temp
    return sum
print(fiboEvenSum(34))

File: project-Euler/javascript/2.js
function fiboEvenSum(n) {
    let a = 1;
    let b = 2;
    let sum = b;
    let next = a+b;
    
    while (next <= n){
      if (next % 2 == 0){
        sum += next;
        temp = next;
        next += b;
        b = temp;
      }else{
        temp = next;
        next += b;
        b = temp;
      }
    
    }
      return sum;
    }
console.log(fiboEvenSum(8))